modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -998,6 +998,18 @@
 				2
 				4
 				[/codeblock]
+				To iterate over an [Array] backwards, use:
+				[codeblock]
+				var array = [3, 6, 9]
+				for i in range(array.size() - 1, -1, -1):
```

Comment:
That's a pretty unwieldy syntax :/
It also invalidates the original description which says that the range goes from (initial, final-1), as if the decrement is negative, it's actually (initial, final+1).

Reply:
Is there a better syntax we could use for this?

And yes, the final value is actually `final + 1` when iterating backwards.

Reply:
I switched the syntax to use a `while` loop as requested by GDQuest.